reconcile/test/ocm/test_utils_ocm_status_board.py

```diff
@@ -0,0 +1,142 @@
+    METADATA_MANAGED_BY_KEY,
+    METADATA_MANAGED_BY_VALUE,
+    create_application,
+    create_product,
+    delete_application,
+    delete_product,
+    get_managed_products,
+    get_product_applications,
+)
+
+
+@pytest.fixture
+def application_status_board() -> list[dict[str, Any]]:
+    return [
+        {
+            "id": "foo",
+            "name": "foo",
+            "fullname": "foo",
+            "metadata": {METADATA_MANAGED_BY_KEY: METADATA_MANAGED_BY_VALUE},
+        },
+        {
+            "id": "bar",
+            "foo": "bar",
+            "metadata": {METADATA_MANAGED_BY_KEY: METADATA_MANAGED_BY_VALUE},
+        },
+        {
+            "id": "oof",
+        },
+    ]
+
+
+@pytest.fixture
+def products_status_board() -> list[dict[str, Any]]:
+    return [
+        {
+            "id": "foo",
+            "name": "foo",
+            "fullname": "foo",
+            "metadata": {METADATA_MANAGED_BY_KEY: METADATA_MANAGED_BY_VALUE},
+        },
+        {
+            "id": "bar",
+            "foo": "bar",
+            "metadata": {METADATA_MANAGED_BY_KEY: METADATA_MANAGED_BY_VALUE},
+        },
+        {
+            "id": "oof",
+        },
+    ]
+
+
+def test_get_product_applications_fields(
+    mocker: MockFixture, application_status_board: list[dict[str, Any]]
+) -> None:
+    ocm = mocker.patch("reconcile.utils.ocm_base_client.OCMBaseClient")
+    mocker.patch(
+        "reconcile.utils.ocm_base_client.OCMBaseClient.get_paginated",
+        return_value=iter(application_status_board),
+    )
+    apps = get_product_applications(ocm, "foo")
+    assert len(apps) == 2
+    assert apps[0] == application_status_board[0]
+    # key foo not in APPLICATION_DESIRED_KEYS
+    assert apps[1] == {
+        "id": "bar",
+        "metadata": {METADATA_MANAGED_BY_KEY: METADATA_MANAGED_BY_VALUE},
+    }
+
+
+def test_get_managed_products(
+    mocker: MockFixture, products_status_board: list[dict[str, Any]]
+) -> None:
+    ocm = mocker.patch("reconcile.utils.ocm_base_client.OCMBaseClient")
+    mocker.patch(
+        "reconcile.utils.ocm_base_client.OCMBaseClient.get_paginated",
+        return_value=iter(products_status_board),
+    )
+
+    products = get_managed_products(ocm)
+    assert len(products) == 2
+    assert products[0] == products_status_board[0]
+    # key foo not in APPLICATION_DESIRED_KEYS
+    assert products[1] == {
+        "id": "bar",
+        "metadata": {METADATA_MANAGED_BY_KEY: METADATA_MANAGED_BY_VALUE},
+    }
+
+
+def test_create_product(mocker: MockFixture) -> None:
+    ocm = mocker.patch("reconcile.utils.ocm_base_client.OCMBaseClient")
+    p = mocker.patch("reconcile.utils.ocm_base_client.OCMBaseClient.post")
+
+    create_product(ocm, {"name": "foo"})
+
+    p.assert_called_once_with(
+        "/api/status-board/v1/products/",
+        data={
+            "metadata": {METADATA_MANAGED_BY_KEY: METADATA_MANAGED_BY_VALUE},
+            "name": "foo",
+        },
+    )
```

Comment:
similarly, 

```python
ocm = mocker.patch("reconcile.utils.ocm_base_client.OCMBaseClient", autospec=True).return_value
```
should be enough, then it can be used like

```python
ocm.post.assert_called_once_with(
```